lib/sysfs_device.c

```diff
@@ -247,7 +247,7 @@ struct sysfs_device *sysfs_open_device_tree(const char *path)
 			if (new == NULL) {
 				dbg_printf("Error opening device tree at %s\n",
 						cur->path);
-			  sysfs_close_device(devlist);
+				sysfs_close_device(devlist);
```

Comment:
Thanks for the patches.  This hunk, should be fixed in previous commit.

Reply:
It looks like I messed up my history when I rebased my changes. Sorry about that. 

Would the best way for me to fix this be to open up a new pull request with the corrected git history and the change mentioned below?

Reply:
Can you please do a git force push to the same pull request.

Reply:
Alright. I corrected the history so that this commit does not show the changes that should be in the last commit.